Sprint-1/implement/sum.js

```diff
@@ -1,4 +1,13 @@
 function sum(elements) {
+  const numericalElements = [];
+  for (const element of elements) {
+    if (typeof element === "number") {
+      numericalElements.push(element);
+    }
+  }
+  return numericalElements.reduce((a, c) => a + c, 0);
```

Comment:
This works, but I think `reduce` is often quite hard to understand and follow. Can you think of a way of implementing this function which _doesn't_ require accumulating an array of values and then using `reduce`?

Reply:
This is simpler, but you're still ending up accumulating an array of values. Can you think of a way to not need a second array at all?